reader.ts

```diff
@@ -189,6 +189,7 @@ export class CSVReader {
   }
 
   private processColumn() {
+    this.emptyLine = false;
```

Comment:
I think it's better to move this to line 362, right after `this.debug("columnSeparator");` to keep `this.emplyLine` mutations within the same method.